js/src/data/actions.js

```diff
@@ -228,6 +228,7 @@ export function* fetchShippingTimes() {
 			return {
 				countryCode: el.country_code,
 				time: Number( el.time ),
+				maxTime: Number( el.max_time ),
```

Comment:
The new property is missing from the `ShippingTime` type.

https://github.com/woocommerce/google-listings-and-ads/blob/c1dffd305b34192e0d6daeb0e36571565c86827d/js/src/data/actions.js#L209-L215